airflow-core/src/airflow/ui/src/pages/Asset/AssetLayout.tsx

```diff
@@ -128,6 +128,26 @@ export const AssetLayout = () => {
           </PanelResizeHandle>
           <Panel defaultSize={30} minSize={20}>
             <Header asset={asset} />
+           {asset?.extra && Object.keys(asset.extra).length > 0 && (
+            <Box mt={3} mb={3} px={3}>
+              <Text fontWeight="bold" mb={2}>
+                 {translate("assets:static_asset_metadata")}
```

Comment:
```suggestion
                 {translate("assets:additional_data")}
```

maybe let's try additional data? the reason why I don't want to use `extra` is because of https://github.com/apache/airflow/issues/55200, but suggestion is welcome. Naming is hard 🤯

Reply:
Thanks, @Lee-W — that makes sense! I’ve updated the label to use Additional Data to stay consistent with the naming direction and avoid confusion with extra. 👍